### DISCUSSION TITLE
ebesucher useragent

### DISCUSSION THREAD

**Discussion by jupitert25** [engageub/InternetIncome#79] [General]
is there any way to change user-agent to window? I tested with "about: config" method and it won't load any website only ran a countdown page so I removed it and worked again. but when I installed the "user-agent switcher addon" it was working but let it run after 1 hour browser reset itself and the addon was gone

**Comment by engageub** [comment]
Hi,
Useragent can be changed using many extensions available. 
The data is deleted after one hour automatically. 
Script can be modified to avoid restart or you can copy the firefox profile data and delete the existing firefoxprofiledata.zip file. 
That way you will have custom profile. You can modify the profile data as you wish and zip it back to the same file so that the data is picked from that file every hour. 

Thank you

**Reply by engageub** [reply]
Hi,
In test branch, you have option to use random display size for firefox.
You can also use chrome browser. Also, it restarts every two hours instead of 1 hour in test branch.
Thank you

**Comment by jupitert25** [comment]
![image](https://github.com/engageub/InternetIncome/assets/151444646/f5133be4-ad90-461f-9647-ffc101b33f3d)
useragent using lastest version
Keep getting this after replacing firefoxprofiledata.zip

**Reply by engageub** [reply]
Hi,
You have to copy content from firefox_profile_data/data folder after modifying the existing data.
To copy only required folders you can refer the test branch firefoxprofiledata.zip folders.
Please find the steps below. 
1. Start the firefox container, 
2. Install the required firefox extensions. 
3. In InternetIncome folder you will find firefox_profile_data/data folder. Go inside the folder and copy all the files and folders required.
4. Zip the folder similar to firefoxprofiledata.zip or replace all the files/folders inside the zip folder.


Thank you

**Reply by jupitert25** [reply]
Thank you so much . you are the best

**Reply by jupitert25** [reply]
sad, yesterday test user agent on the addon working now ebesucher detected and only ran countdown page

**Reply by engageub** [reply]
It depends on IP address also. If you are using random useragent, any useragent can appear.
May be it was timing when the webpage actually was payable. 
They may not pay again for the same webpage. 
Its not recommended to spoof the actual browser. They pay for testing the websites in the browser for marketing purposes.
If you wish to use chrome browser, you can use test branch.
You can create custom profiles as you wish and play around with it.

Thank you

**Reply by jupitert25** [reply]
I test Chrome browser too and it's not working even no useragent switcher . ebesucher is hard to bypass

**Reply by engageub** [reply]
They may be detecting using IP address and if there are no new websites to display, earnings may not be visible.

**Reply by engageub** [reply]
Also, please note that this is not a bypass script. Official extension and real firefox browser is being used.

Thank you

**Reply by jupitert25** [reply]
you are right. I tried different IP and found one working

**Reply by engageub** [reply]
Hello,
The code has been updated for random display now. The minimum recommended size for display is1280 x 1024

https://www.ebesucher.de/faq/wie-kann-ich-mehr-verdienen

Thank you
